app/controllers/student_identifier_types_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+class StudentIdentifierTypesController < ApplicationController
+  include OrganizationAuthorization
+
+  before_action :check_flipper
+  before_action :save_referer, only: [:new]
+
+  def index
+    @student_identifier_types = @organization.student_identifier_types
+  end
+
+  def new
+    @student_identifier_type = StudentIdentifierType.new
+  end
+
+  def create
+    @student_identifier_type = StudentIdentifierType.new(student_identifier_type_params)
+    if @student_identifier_type.save
+      flash[:success] = "\"#{student_identifier_type.name}\" has been created!"
+      redirect_to session.delete(:return_to) || { action: 'index' }
```

Comment:
Just out of curiosity can you describe the scenario where we set a `:return_to` for the user?